LICENSE

```diff
@@ -181,6 +181,33 @@ Neovim's license follows:
       incurred by, or claims asserted against, such Contributor by reason
       of your accepting any such warranty or additional liability.
 
+	 END OF TERMS AND CONDITIONS
```

Comment:
All it means is the end of terms and conditions. The appendix is still part of the license.

Even see http://choosealicense.com/licenses/apache-2.0/

It says to copy the entire text into `LICENSE`.